client/src/components/ImageUploadContainer.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { FormattedMessage } from 'react-intl';
+import { uploadImage, fetchImages } from '../store/actions/ImagesActions';
+
+interface State {
+  image: File | null;
+}
+
+interface Props {
+  uploadImage(image: File): Promise<void>;
+  fetchImages(): Promise<void>;
+}
+
+class ImageUploadContainerView extends React.PureComponent<Props, State> {
+  state: State = { image: null };
+  triggerUpload = (event: React.MouseEvent) => {
```

Comment:
We don't use event variable so no need to declare it.

Reply:
fixed via https://github.com/cverna-hackathons/sup/pull/27/commits/846ec703b2b2f4b3f79d7ca27955d0e9c7fe92e2